src/docs/Foundations/Utilities/Images/code.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Code'
+order: 2
+---
+
+> The implementation of Mozaic ratios in your code can be done in two ways:
+>
+> 1. By using the `set-ratio` mixin
+> 2. By using ratio helpers
+>
+> The above documentation documents both of these approaches.
```

Comment:
The following documentation represents both of these approaches.

Reply:
fixed